### ISSUE TITLE
Invocations should be placed indented and not outdented

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#1321]
We should report an issue with following code:

```c#
public static async Task Main(string[] args)
  {
      await CreateHostBuilder(args)
          .RunConsoleAsync();
  }
```

The `.RunConsoleAsync()` invocation should be placed indented to `CreateHostBuilder` and not outdented.

**Comment by RalfKoban** [comment]
The indentation fix should align it 4 characters from the right, before the `(` opening brace.
Be aware of short names like `Is()`. In such case it should align to the left side, to avoid an outdented invocation.
